train_vnext/training.py

```diff
@@ -13,10 +13,10 @@
 
 CONFIG_PATH = os.environ.get('ALCONFIG', None)
```

Comment:
@samiyaakhtar we need to the [legacy config ](https://github.com/CatalystCode/active-learning-detect/blob/master/config.ini). Maybe we can add something like
```
LEGACY_CONFIG_PATH = os.environ.get('ALCONFIG_LEGACY', None)
```